server/chat-commands/core.ts

```diff
@@ -370,7 +370,8 @@ export const commands: ChatCommands = {
 		if (this.cmd === 'mee' && /[A-Z-a-z0-9/]/.test(target.charAt(0))) {
 			return this.errorReply(this.tr`/mee - must not start with a letter or number`);
 		}
-		target = this.checkChat(`/${this.cmd} ${target || ''}`);
+		(target as string | null) = this.checkChat(`/${this.cmd} ${target || ''}`);
+		if (!target) return;
```

Comment:
This doesn't return `null` anymore; we refactored it to throw `Chat.ErrorMessage`, remember? That was what I meant by "???".